### PR TITLE
Prepare ProxySQL with multi-arch

### PR DESCRIPTION
**Pull request by tamalsaha**
Signed-off-by: Tamal Saha <tamal@appscode.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+  && apt-get install -y --no-install-recommends apt-transport-https ca-certificates default-mysql-client \
+  && apt-get purge -y --auto-remove ca-certificates \
+  && rm -rf /var/lib/apt/lists/* /usr/share/doc /usr/share/man /tmp/*
 
-ENTRYPOINT ["/tini","-g","--"]
-CMD ["/entrypoint.sh"]
+COPY proxysql.cnf /etc/proxysql.cnf
+COPY scripts      scripts
+COPY sql          sql
+COPY --from=0 /tini /tmp/scripts/tini
```
Comment (spectro30) [inline Dockerfile]: add newline

File: scripts/configure-proxysql.sh [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+
+# wait for proxysql process to be run
+wait_for_mysql admin admin 127.0.0.1 6032
+
+mysql_exec \
+    $PROXYSQL_ADMIN_USER \
+    $PROXYSQL_ADMIN_PASSWORD \
+    127.0.0.1 \
+    6032 \
+    "$cleanup_sql $hostgroups_sql $servers_sql $users_sql $queries_sql" \
```
Comment (spectro30) [inline scripts/configure-proxysql.sh]: what's the function of cleanup_sql?

if not implemented yet, use that for cleaning previous proxy related config settings like monitor user.

File: scripts/configure-proxysql.sh [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+"
+    echo $sql
+}
+
+proxycluster_sql=$(get_proxyservers_sql)
+
+log "INFO" "sql query to configure proxysql cluster
+$proxycluster_sql"
+
+if [ $DO_CLUSTER == "true" ]; then
```
Comment (spectro30) [inline scripts/configure-proxysql.sh]: `PROXY_CLUSTER` sounds more relatable than `DO_CLUSTER`

File: scripts/configure-proxysql.sh [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+    if [[ "$PRE_CLUSTER_PASS" == "" ]]; then
+      mysql -uadmin -padmin -h127.0.0.1 -P6032 -Nbe "set admin-cluster_password='$CLUSTER_PASSWORD';"
+    else
+      mysql -uadmin -padmin -h127.0.0.1 -P6032 -Nbe "set admin-cluster_password='$PRE_CLUSTER_PASS;$CLUSTER_PASSWORD';"
+    fi
+
+
+    mysql -uadmin -padmin -h127.0.0.1 -P6032 -Nbe "SAVE ADMIN VARIABLES TO DISK;"
+    mysql -uadmin -padmin -h127.0.0.1 -P6032 -Nbe "LOAD ADMIN VARIABLES TO RUNTIME;"
+fi
```
Comment (spectro30) [inline scripts/configure-proxysql.sh]: add newline

**Review by spectro30 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (TasdidurRahman) [reply Dockerfile]: added

File: scripts/configure-proxysql.sh (hunk 4 above)
Comment (TasdidurRahman) [reply scripts/configure-proxysql.sh]: added

File: scripts/configure-proxysql.sh (hunk 2 above)
Comment (TasdidurRahman) [reply scripts/configure-proxysql.sh]: removed

File: scripts/configure-proxysql.sh (hunk 3 above)
Comment (TasdidurRahman) [reply scripts/configure-proxysql.sh]: changed env name

**Review by spectro30 — APPROVED** [review]
LGTM

**Pull request merged**